### ISSUE TITLE
Turn exercise 011 into practice

### ISSUE THREAD

**Issue by cuihtlauac** [ocaml/ocaml.org#2179]
Extract the OCaml code from the markdown file corresponding to this exercise. The file is: `data/exercise/011_NAME.md` (where `011` is the number from the title of this issue and `NAME` is the rest of the file name, for instance `data/exercise/001_tail.md` is the file which corresponds to exercise `001`). Use the code in this file to create a folder `practice/011` (using this same number).

Populate the `011` folder with (almost) the same files as in folders `practice/001` and `practice/002` (they are examples).

Three files are different in each folder:
1. `practice/011/ex.ml`: it contains the function's signature, which needs to be implemented by the learner; 
1. `practice/011/work/impl.ml`: it includes the template meant to be filed by the learner;
1. `practice/011/answer/impl.ml`: consists of the solution from the original markdown file.

The examples included in the original markdown file must be turned into test cases in the `practice/011/ex.ml` file.

Folder structure to create:
```
practice/011/
├── answer
│   ├── impl.ml
│   └── test
│       ├── dune
│       └── run.ml
├── dune
├── dune-project
├── ex.ml
└── work
    ├── impl.ml
    └── test
        ├── dune

**Comment by Kxrishx03** [comment]
Hii! @cuihtlauac Can I take part in it?

**Comment by cuihtlauac** [comment]
@Kxrishx03 pick a single issue to start with

**Comment by Kxrishx03** [comment]
@cuihtlauac Sorry wasn't aware about that, will keep that in mind next time.

**Comment by divyankachaudhari** [comment]
@cuihtlauac I'd like to work on this

**Comment by cuihtlauac** [comment]
Fixed by #2196
